docs/appendices/release-notes/5.4.4.rst

```diff
@@ -94,25 +88,25 @@ Fixes
   of object types created by ``CREATE TABLE AS`` to be overridden to
   ``strict``.
 
- - Fixed an issue that caused ``RETURNING`` clause, referring to
-   non-deterministic ``GENERATED`` or ``DEFAULT`` columns, return a value, not
-   equal to the actually persisted one.
+- Fixed an issue that caused ``RETURNING`` clause, referring to
+  non-deterministic ``GENERATED`` or ``DEFAULT`` columns, return a value, not
+  equal to the actually persisted one.
 
- - Fixed an issue that caused storing inconsistent values on primary and
-   replica for ``GENERATED`` or ``DEFAULT`` columns, using non-deterministic
-   functions, returning different values even within the scope of a single
-   query. e.g.:: ``gen_random_text_uuid`` and ``random``. Functions like
-   ``NOW`` or ``CURRENT_TIMESTAMP`` are unaffected by this issue.
+- Fixed an issue that caused storing inconsistent values on primary and
```

Comment:
Applying https://github.com/crate/crate/pull/14883, otherwise https://cratedb.com/docs/crate/reference/en/5.4/appendices/release-notes/5.4.4.html has vertical lines next to those entries.